app/controllers/users_controller.rb

```diff
@@ -0,0 +1,33 @@
+class UsersController < ApplicationController
+
+  def index
+      @users = User.paginate(page: params[:page], per_page: 20)
```

Comment:
small nitpick, but you have 2 spaces for indentation everywhere else and 4 spaces here, consistency is golden